src/core_plugins/kibana/public/discover/components/field_chooser/discover_field.js

```diff
@@ -95,6 +93,13 @@ app.directive('discoverField', function ($compile) {
         }
       };
 
+      $scope.onKeyUp = function onKeyUp(e, field) {
+        // Support keyboard accessibility by toggling display on ENTER or SPACE keypress.
+        if ([13, 32].includes(e.keyCode)) {
+          $scope.toggleDetails(field);
+        }
+      };
```

Comment:
If we're going to need to do this often, do you think it would be worthwhile to create a custom attribute directive called `kbn-buttonPress` or something like that, which could handle this keycode checking and also do the preventDefault for the space bar on keyDown? It might make this a bit easier to implement in other components.